jolt-core/src/main/java/com/bazaarvoice/jolt/utils/JoltUtils.java

```diff
@@ -62,6 +62,54 @@ public static void removeRecursive( Object json, String keyToRemove ) {
     }
 
     /**
+     * Replaces a value recursively from anywhere in a JSON document.
+     * NOTE: mutates its input.
+     *
+     * @param json        the Jackson Object version of the JSON document
+     *                    (contents changed by this call)
+     * @param keyToReplace the key to remove from the document
+     * @param value        the new value for the keyToReplace
+     */
+    public static void replaceRecursive( Object json, String keyToReplace, Object value ) {
```

Comment:
it is more of an "updateValueRecursive"  or "replaceValueRecursive"

Reply:
Renamed to replaceValueRecursive